bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckCommand.java

```diff
@@ -0,0 +1,41 @@
+package org.openjdk.skara.bots.pr;
+
+import org.openjdk.skara.forge.HostedRepositoryPool;
+import org.openjdk.skara.forge.PullRequest;
+import org.openjdk.skara.forge.PullRequestUtils;
+import org.openjdk.skara.issuetracker.Comment;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+import java.util.HashSet;
+import java.util.List;
+
+public class CheckCommand implements CommandHandler {
+    @Override
+    public String description() {
+        return "run Jcheck again";
+    }
+
+    @Override
+    public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath, CommandInvocation command, List<Comment> allComments, PrintWriter reply) {
+        try {
+            var workItem = new CheckWorkItem(bot, pr, e -> {});
```

Comment:
Don't think you actually have to do the check here, `CommandWorkItem` will schedule an additional check after every command anyway. But to force the check to run you'll need to ensure that the "metadata" of the existing check becomes invalid. If you update `metadataComments` to add a special reply marker from the `/check` command, and ensure that the check command outputs it, I think thinks should work.